### PR TITLE
Automate testing and building of deb package using GitHub Actions

### LINKED ISSUE #1: Make a .deb package to automate the long installation
The current installation process is really tedious.

### PR DESCRIPTION
**Pull request by DrauzJu**
Aims to close #1 

With this PR, a GitHub Actions Workflow is introduced, which is triggered on every push to the master branch. It sets up the environment (Ubuntu 18.04), runs the tests, builds a deb-package and uploads it. A demo of a successful run including the produced deb-package can be found [here](https://github.com/DrauzJu/zeyple/commit/576917047a2557bd799367defe06c05ca07c4eb3/checks?check_suite_id=262492639).

As a next step, a test matrix could be introduced to test gpg and gpgme. Furthermore, it should be automatically tested if the generated deb-pacakge can be installed successfully.

### CODE REVIEW

File: .github/workflows/zeyple.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 2.7
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ruby ruby-dev rubygems build-essential gnupg python-gpg libgpgme-dev swig
```
Comment (infertux) [inline .github/workflows/zeyple.yml]: What do we need the `swig` package for?

**Comment by infertux** [comment]
This looks great! Thank you for setting that up. And I'm glad to see this in the build output:

`DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date.`

I can't wait to drop support for Python 2 :D

File: .github/workflows/zeyple.yml (hunk 1 above)
Comment (DrauzJu) [reply .github/workflows/zeyple.yml]: It seems that the `gpg` Python dependency needs it: https://github.com/DrauzJu/zeyple/commit/1344e64f7d5a4815e7ff61b71be6b8109293d45b/checks?check_suite_id=262393072#step:4:328

**Comment by DrauzJu** [comment]
Thanks for merging! I think you need to register here to enable GitHub Actions in the repository: https://github.com/features/actions

**Comment by infertux** [comment]
Thanks @DrauzJu! It's now live at https://github.com/infertux/zeyple/actions :smile:

**Pull request merged**